frontend/contexts/AuthContext.tsx

```
@@ -0,0 +1,190 @@
+      
+      // Store tokens and user data
+      setAccessToken(response.data.access_token);
+      setRefreshToken(response.data.refresh_token);
+      setUser(response.data.user);
+      setIsAuthenticated(true);
+      
+      // Save refresh token securely
+      await SecureStore.setItemAsync('refreshToken', response.data.refresh_token);
+    } catch (error) {
+      // Re-throw the error so the UI can handle it
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)